### ISSUE TITLE
sugestion

### ISSUE THREAD

**Issue by HERRKIN** [sdrelton/cryptocoin-indicator#2]
I think there is no need for a logo at the indicator level, what I would like instead is a configuration where I could choose how many pairs I want displayed, I would want to see btc and ltc too at the same time without having to open a new instance.

**Comment by sdrelton** [comment]
Yeah I've also thought about adding support for two or more indicators at the same time. It's something I'd like to get around to soon.

Sent from my Samsung Galaxy smartphone.

-------- Original message --------
